### ISSUE TITLE
Mandatory to adopt Decidim's social contract?

### ISSUE THREAD

**Issue by bzg** [Platoniq/decidim-install#30]
I have read Decidim's [social contract](https://docs.decidim.org/social-contract/en/social-contract/) and there is this sentence:

> The use and development of the Decidim platform by an institution or social group of any kind involves the full and integral acceptance of, and commitment to, this social-contract.

I am trying to understand the scope of "the use and development of the Decidim platform".

Does it mean that I cannot install Decidim software without adopting Decidim's social contract?
Does it also mean that I cannot provide a contribution to Decidim software without adopting the social contract? Or is the social contract just for those who _use_ the https://decidim.org instance?

Thanks for any clarification!

**Comment by microstudi** [comment]
Hi,
the use of the software is covered by the AGPL license, you are not forced to any other constraint regarding its use apart from the ones covered there. Same as any other open source software, just follow the license.
However, you have to commit to the Social Contract if you want to form part of the Decidim Association, participate in it, etc. Decidim may (note the "may") not accept contributions from people not following the same principles. It's more about ethics and social justice and having a community with the same goals, but no one is going to stop you using Decidim in any way you want.

Bottom line, you can install it, use it, experiment it and contribute to it. If we discover then that you are using it to sell guns to children we may ban you from any further contributions 😉

**Comment by bzg** [comment]
Hi @microstudi and thanks very much for your reply!

> the use of the software is covered by the AGPL license, you are not forced to any other constraint regarding its use apart from the ones covered there. Same as any other open source software, just follow the license.

That's very reassuring. The sentence I quoted above from your Social Contract says that the "use and development of the Decidim platform" makes it mandatory to endorse the social contract, and it is not clear whether the "platform" refers to decidim.org or to the software behind. There is also this sentence in the FAQ:

> You can use it as you want in your organization, as long as you respect our Social Contract.

I guess "it" refers to decidim.org as a web service, not to the software itself.

I suggest replacing this sentence from the Social Contract

> "The use and development of the Decidim platform by an institution or social group of any kind involves the full and integral acceptance of, and commitment to, this social-contract."

by this one:

> The Decidim software is covered by the AGPL license. [You can use it, modify it and redistribute derived versions of it you respect the AGPL license.] Contributions to the Decidim software are moderated by the Decidim association. You are welcome to take part of our association if you accept our Social Contract. Similarily, using the decidim.org platform requires you to respect our Social Contract.

The part within square brackets is perhaps not necessary, but clarifying the distinction between the software, the decidim.org platform and the governance of contributions to the software through the Decidim association would surely help avoiding misinterpretation.

My reaction comes from reading [this french article](https://scinfolex.com/2019/05/09/le-contrat-social-de-decidim-vers-des-logiciels-libres-a-mission/) where the author asserts that Decidim's social contract differs from Debian's social contract by imposing users of the software to force users of their instances to follow the social contract too.

What do you think?

**Comment by microstudi** [comment]
well you've raised a good debate here.
I'm going to ask this into the meta decidim community, which by the way I invite to join and express yours concerns. I'll share the link here so we may discuss it further there.
Personally, I agree with your proposed changes in the text and I think this aspect should be clarified.

Strictly legally speaking though, I think the only thing that really matters is the AGPL, otherwise they should have created a new license.

**Comment by bzg** [comment]
Thanks! I'm glad if I can help clarifying a few things here.  My position in a nutshell:

- I like Decidim's social contract;
- I find it useful to have such a contract for the governance of the Decidim free software;
- I think the TOS of the decidim.org platform should use a subset of the social contract, namely the one relevant to everything except software contribution (transparency, etc.);
- It is fine to encourage Decidim users to (1) contribute back by committing to the social contract when they want to contribute to the software and to (2) use similar TOS than the decidim.org platform when they want to implement their own...
- ... but none of the two options above should be mandatory.

In general, while I welcome the idea of creating a digital commons by connecting with likeminded groups, I would find it very confusing and somewhat dangerous to "limit" the use of the software, making it _de facto_ a non-free software.

**Comment by microstudi** [comment]
Regarding this, I've started a threat in the Meta Decidim community about what are you proposing.

You can find it here: https://meta.decidim.org/processes/roadmap/f/122/proposals/14607?locale=en

It would be preferable to transfer this discussion and it's conclusions there. If you could join the Meta Decidim community and express your thoughts there would be great!

Thanks!

**Comment by bzg** [comment]
Thanks a lot for following this up. I will join the discussion on https://meta.decidim.org if more input in required.

**Comment by bzg** [comment]
The discussion seems to stall. Do you know if anything is planed to make progress? Sorry to use you as a proxy, but I'm reluctant to create a new account and dive into the conversation there.

**Comment by microstudi** [comment]
No worries. In fact discussion has ended and the social contract will be modified to be more precise. 
We still need to decide the changes exactly. Here is the guy who actually wrote the social contract.: 

![image](https://user-images.githubusercontent.com/1401520/60169971-115d5380-9808-11e9-9ec6-6d7c0e838648.png)

So, you can use the software as any other AGPL. Social contract will only apply to the Decidim organization and as way to encourage good practices to whoever is using Decidim.

**Comment by bzg** [comment]
Thanks for the quick feedback! I'll stay tuned.

**Comment by microstudi** [comment]
Happy to say that this matter has been resolved.
It should be clear now that the social contract is a code of conduct for the Decidim community and the software is under the AGPL. In practical terms that means that you can do whatever you want with the software (if complies the AGPL) but the Decidim community might not accept changes to main repository that not comply with the social contract (in that case you will probably be suggested to create a separate module to add that functionality).

**Comment by bzg** [comment]
Yes, it is clear now, thanks again for handling this complex issue. Closing it!
